OJP/OJP_Trips.xsd

```diff
@@ -743,6 +743,38 @@
 					<xs:documentation>Number how often the access feature occurs in this PathLink</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="From" type="PathLinkEndStructure" minOccurs="0">
```

Comment:
This structure makes it possible to only supply a `From`, but no `To`. Maybe add a group consisting of exactly one `From` and one `To` element and give the group `minOccurs="0"` instead?

Reply:
@sgrossberndt  The structure also makes it possible to supply only a *To*. But wouldn't that be welcome? - When giving directions, it is usually sufficient to provide information only for the *To* (for instance the PublicCode of the target level when using an elevator). 
Regarding your previous comment: The idea of the Id was not to point to a *Level*, but more generally to an element at the end of a PathLink - for instance a NeTEx AccessSpace or an EquipmentPlace (this would later allow for retrieving additional information, e.g., the coordinates of the EquipmentPlace of the elevator). Thus, *ToLevel*->*Id* would not be what was intended. That said, I think it is better to stay with the proposed solution. What do you think?